docs/methodology.md

```diff
@@ -1,48 +1,52 @@
 # Methodology
+
 Below we describe the methodology used to benchmark [CLP] and other tools.
 
 ## Test datasets
+
 There are two types of logs used in the benchmark:
-+ **Unstructured logs** are log events that don't follow a predefined format or structure, making it
+
+- **Unstructured logs** are log events that don't follow a predefined format or structure, making it
   difficult to automatically parse or analyze them. Unstructured logs are often just text
-  interspersed with variable values like the log event's timestamp, log-level, and others such as
+  interspersed with variable values like the log event's timestamp, log level, and others such as
   usernames, etc. For example:
+
   ```
   2018-06-05 06:15:29,701 DEBUG org.apache.hadoop.util.Shell: setsid exited with exit code 0
   ```
 
-+ **Semi-structured logs** do have follow a predefined format, making them parseable, but their
+- **Semi-structured logs** do have follow a predefined format, making them parseable, but their
```

Comment:
Let's call them dynamically-structured rather than semi-structured. That's the technical term that we've decided to use.